script/model/training_util/data_transforming_util.py

```diff
@@ -66,13 +66,13 @@ def _tuple_num_linear_log_predict_transform(x, y):
 
 def _tuple_num_log_train_transform(x, y):
     # Transform down the target according to the log tuple num value in the input
-    tuple_num = np.copy(x[:, data_info.TUPLE_NUM_INDEX])
+    tuple_num = np.copy(x[:, data_info.INPUT_CSV_INDEX[ExecutionFeature.EST_OUTPUT_ROWS]])
     return y / (np.log2(tuple_num) + 1e-6)[:, np.newaxis]
```

Comment:
Change the `1e-6` to `1` here. This is a minor model bug fix I have in my branch. But I anticipate that my branch will take even more time to be eventually merged than your branch, so let's just fix it here.